gui/src/components/settings/pages/AdvancedSettings.tsx

```diff
@@ -0,0 +1,187 @@
+import { useLocalization } from '@fluent/react';
+import { useState } from 'react';
+import { Typography } from '@/components/commons/Typography';
+import {
+  SettingsPageLayout,
+  SettingsPagePaneLayout,
+} from '@/components/settings/SettingsPageLayout';
+import { WrenchIcon } from '@/components/commons/icon/WrenchIcons';
+import { Button } from '@/components/commons/Button';
+import { SettingsResetModal } from '@/components/settings/SettingsResetModal';
+
+import { invoke } from '@tauri-apps/api/core';
+import { open } from '@tauri-apps/plugin-shell';
+import { error } from '@/utils/logging';
+import { defaultConfig as defaultGUIConfig, useConfig } from '@/hooks/config';
+import { defaultValues as defaultDevConfig } from '@/components/widgets/DeveloperModeWidget';
+import { RpcMessage, SettingsResetRequestT } from 'solarxr-protocol';
+import { useWebsocketAPI } from '@/hooks/websocket-api';
+
+function guiDefaults() {
+  // Destructure the properties to exclude "lang"
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { lang, ...guiDefaults } = defaultGUIConfig;
+
+  // Include "devSettings" which has all the properties of "defaultDevConfig"
+  // @ts-expect-error "devSettings" is not in the "guiDefaults" object but we want to include it (from "defaultDevConfig")
+  guiDefaults.devSettings = defaultDevConfig;
+
+  return guiDefaults;
+}
+
+export function AdvancedSettings() {
+  const { l10n } = useLocalization();
+  const { setConfig } = useConfig();
+
+  const [skipWarningGui, setSkipWarningGui] = useState(false);
+  const [skipWarningServer, setSkipWarningServer] = useState(false);
+  const [skipWarningAll, setSkipWarningAll] = useState(false);
+  const { sendRPCPacket } = useWebsocketAPI();
+
+  const openConfigFolder = async () => {
+    try {
+      const configPath = await invoke<string>('get_config_path');
+      await open('file://' + configPath);
```

Comment:
this should work according to the docs
```suggestion
      await open(configPath);
```

Reply:
See `tauri.conf.json`'s comment